src/code/z_parameter.c

```diff
@@ -2178,10 +2177,9 @@ s32 Health_ChangeBy(GlobalContext* globalCtx, s16 healthChange) {
                  gSaveContext.healthCapacity);
 
     // clang-format off
-    // NOLINTNEXTLINE
-    if (healthChange > 0) Audio_PlaySoundGeneral(NA_SE_SY_HP_RECOVER, &D_801333D4, 4,
-                                                 &D_801333E0, &D_801333E0, &D_801333E8);
-    else if ((gSaveContext.doubleDefense != 0) && (healthChange < 0)) {
+    if (healthChange > 0) { Audio_PlaySoundGeneral(NA_SE_SY_HP_RECOVER, &D_801333D4, 4,
+                                                   &D_801333E0, &D_801333E0, &D_801333E8);
+    } else if ((gSaveContext.doubleDefense != 0) && (healthChange < 0)) {
         healthChange >>= 1;
```

Comment:
I'm guessing /= 2 doesn't work here?  I think it would make more sense than >>= if possible.

Reply:
It does not. And it's actually not equivalent because healthChange is signed, and a negative division is not treated as a straight arithmetic shift by IDO.